pkgs/apps/exciting/default.nix

```diff
@@ -1,6 +1,6 @@
 { lib, stdenv, fetchFromGitHub
 , gfortran, perl, libxslt, runtimeShell
-, blas, lapack, mpi, arpack }:
+, blas, lapack, mpi, arpack, sgroup }:
```

Comment:
I wonder if this should be optional? It turns exciting into a package that needs  unfree software to be built.

Reply:
I can make it optional, yes. I've kept "unfree" in sgroup to be safe. In fact, they do not specify any license or copyright. :grimacing:

Reply:
On the `homepage` they refer to the [CPC license](https://beta.elsevier.com/about/policies-and-standards/open-access-licenses/elsevier-user/cpc), which reads like "unfree" to me. I think the classification as unfree is appropriate here.

Cite from the CPC license:
1. This licence entitles the licensee (one person) and the licensee's research group to obtain a copy of the source or executable code and to use the acquired program for academic or non-profit use within a research group; or, it entitles the licensee (one company, organisation or computing centre) to install the program and allow access to the executable code to members of the licensee's organisation for academic or non-profit use. **No user or site will re-distribute the source code or executable code to a third party in original or modified form without the written permission of the author**.